src/components/Facility/Consultations/VentilatorPlot.tsx

```
@@ -28,222 +31,48 @@
-  const dates = Object.keys(results).map((p: string) => formatDateTime(p));
+    const fetchDailyRounds = async (
+      currentPage: number,
+      consultationId: string,
+    ) => {
+      const { res, data } = await request(routes.dailyRoundsAnalyse, {
+        body: { page: currentPage, fields: VentilatorPlotFields },
+        pathParams: {
+          consultationId,
+        },
+      });
+      if (res && res.ok && data) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)